orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -718,7 +715,76 @@ TEST(GradientCheckerTest, ConcatGrad) {
     TensorShape x2_shape({2, 4});
     TensorShape y_shape({2, 6});
     gradient_checker.ComputeGradientError(op_def, {x1_shape, x2_shape}, {y_shape}, &max_error,
-                                          {MakeAttribute("axis", int64_t(-1))});
+                                          {MakeAttribute("axis", int64_t(-1))}, true, false);
+    EXPECT_IS_TINY(max_error);
+  }
+}
+
+TEST(GradientCheckerTest, ConcatTrainingGrad) { /*also test w/o shape inferencing */
+  float max_error;
```

Comment:
this code looks very similar to the previous test. can it be refactored into a test helper function?